doc/source/train/examples/intel_gaudi/llama_pretrain.ipynb

```diff
@@ -0,0 +1,568 @@
+    "execution_mode = \"lazy\"\n",
+    "os.environ[\"PT_HPU_LAZY_MODE\"] = \"1\" if execution_mode == \"lazy\" else \"0\"\n",
+    "\n",
+    "main(num_workers=4, execution_mode=execution_mode)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Possible outputs\n",
+    "\n",
+    "``` bash\n",
+    "\n",
+    "...\n",
+    "(RayTrainWorker pid=359077) ============================= HABANA PT BRIDGE CONFIGURATION =========================== \n",
+    "(RayTrainWorker pid=359077)  PT_HPU_LAZY_MODE = 1\n",
+    "(RayTrainWorker pid=359077)  PT_RECIPE_CACHE_PATH = \n",
+    "(RayTrainWorker pid=359077)  PT_CACHE_FOLDER_DELETE = 0\n",
+    "(RayTrainWorker pid=359077)  PT_HPU_RECIPE_CACHE_CONFIG = \n",
+    "(RayTrainWorker pid=359077)  PT_HPU_MAX_COMPOUND_OP_SIZE = 9223372036854775807\n",
+    "(RayTrainWorker pid=359077)  PT_HPU_LAZY_ACC_PAR_MODE = 1\n",
+    "(RayTrainWorker pid=359077)  PT_HPU_ENABLE_REFINE_DYNAMIC_SHAPES = 0\n",
+    "(RayTrainWorker pid=359077) ---------------------------: System Configuration :---------------------------\n",
+    "(RayTrainWorker pid=359077) Num CPU Cores : 152\n",
+    "(RayTrainWorker pid=359077) CPU RAM       : 1056440348 KB\n",
+    "(RayTrainWorker pid=359077) ------------------------------------------------------------------------------\n",
+    "\n",
+    "...\n",
+    "\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.975e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.39, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\\\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.9500000000000004e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.4, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.9250000000000004e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.42, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.9e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.42, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.875e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.4, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.85e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.4, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\n",
+    "(RayTrainWorker pid=339380) {'loss': nan, 'grad_norm': nan, 'learning_rate': 4.825e-05, 'epoch': 0.0, 'memory_allocated (GB)': 40.45, 'max_memory_allocated (GB)': 93.68, 'total_memory_available (GB)': 94.62}\n",
```

Comment:
seems that the loss and grad norm are `nan`, can you try to fix this?

Reply:
OK, fixed this bug, due to the dataset not processed correctly.